source/back.tex

```diff
@@ -18,6 +18,14 @@ \chapter{Bibliography}
     Programming languages, their environments, and system software interfaces ---
     Floating-point extensions for C --- Part 3: Interchange and extended types}
 % Other international standards.
+\item
+  IANA Character Sets Database.
+  Available from:\newline
+  \url{https://www.iana.org/assignments/character-sets/}, 2021-04-01
+\item
+  Unicode Character Mapping Markup Language [online].
+  Edited by Mark Davis and Markus Scherer. Revision 5.0.1; 2017-05-31
+  Available from: \url{http://www.unicode.org/reports/tr22/tr22-8.html}
```

Comment:
I think the two IANA things want to be together. Should we sort "other standards" alphabetically?

Reply:
One would think so, but not knowing how the Bibliography is supposed to be ordered I followed what the paper did.

Added in a separate commit.